linked_list/main.py

```diff
@@ -62,34 +63,94 @@ class LinkedList(AbstractLinkedList):
     """
 
     def __init__(self, elements=None):
-        pass
+
+        self.start = None
+        self.end = None
+            
+        if elements:
+            for element in elements:
+                self.append(element)
 
     def __str__(self):
-        pass
+        return '[' + ', '.join(str(i) for i in self) + ']'
```

Comment:
More pythonic ;-):
```
return '[{}]'.format(', '.join([str(i) for i in self]))
```
Concatenating strings with `+` is not very used, `.format()` is usually preferred.